bibliopixel/util/colors/arithmetic.py

```diff
@@ -1,10 +1,16 @@
 def color_blend(a, b):
-    """Performs a Screen blend on RGB color tuples, a and b"""
+    """
+    Performs a Screen blend on RGB color tuples, a and b
+    DEPRECATED:  use numpy
```

Comment:
Show example or provide link. This is useless to someone who doesn't know numpy.

Reply:
Actually - I'm going to eliminate these files from the .rst build entirely!